packages/react-email/preview/src/components/code-container.tsx

```diff
@@ -72,19 +74,34 @@ export const CodeContainer: React.FC<Readonly<CodeContainerProps>> = ({
       }}
     >
       <div className="h-9 border-b border-slate-6">
-        <div className="py-[10px] px-4 text-xs flex gap-8">
-          {markups.map(({ language }) => {
-            return (
-              <div key={language}>
-                <button
-                  className={`${activeTab !== language && 'opacity-25'}`}
+        <div className="flex">
+          <AnimateSharedLayout>
+            {markups.map(({ language }) => {
+              const isHovered = hovered === language;
+              return (
+                <motion.button
+                  className={`relative py-[10px] px-4 text-xs transition ease-in-out duration-200 ${
+                    activeTab !== language ? 'text-slate-10' : 'text-slate-12'
```

Comment:
Text should be `slate-11` or `slate-12`.

```suggestion
                    activeTab !== language ? 'text-slate-11' : 'text-slate-12'
```